source/OVERVIEW.rst

```diff
@@ -21,13 +21,20 @@ Jupyter Notebooks and Markdown files into publishable content.
 Providing online-executable runtimes and a powerful data caching system, NeuroLibre makes a **groundbreaking preprint
 server where you can plant the seeds of living publications.**
 
+.. seealso:: If you would like to submit a NeuroLibre preprint, please read our `submission guidelines <https://docs.neurolibre.com/en/latest/SUBMIT.html>`_.
+
+As an author and before submitting, you:
```

Comment:
Before submitting: 

should be enough.

Reply:
Note that this was your text haha. Anyway just changes it